release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -151,7 +151,189 @@ For more information, see xref:../openshift_images/image-streams-manage.adoc#ima
+** OpenShift Elasticsearch Operator
+** Service Binding Operator
+** Vertical Pod Autoscaler Operator
+* The following Multus CNI plugins are supported:
+** Bridge
+** Host-device
+** IPAM
+** IPVLAN
+* Alternate authentication providers
+* Automatic Device Discovery with Local Storage Operator
+* CSI Volumes
+** Cloning
+** Expansion
+** Snapshot
+* Cron jobs
+* Descheduler
+* Encrypting data stored in etcd
+* Helm
+* Horizontal pod autoscaling
+* IPv6
+* Monitoring for user-defined projects
+* Multipathing
+* Operator API
+* OC CLI plugins
+* Persistent storage using iSCSI
+* Persistent storage using local volumes (Local Storage Operator)
+* Persistent storage using hostPath
+* Persistent storage using Fibre Channel
+* Persistent storage using Raw Block
+* PodDisruptionBudget
+* OVN-Kubernetes, including IPsec encryption
+* Scheduler profiles
+* Stream Control Transmission Protocol (SCTP)
+* Support for multiple network interfaces
+* Three-node cluster support
+* z/VM Emulated FBA devices on SCSI disks
+* 4K FCP block device
+
+These features are available only for {product-title} on {ibmzProductName} and {linuxoneProductName} for {product-version}:
+
+* HyperPAV enabled on {ibmzProductName} and {linuxoneProductName} for the virtual machines for FICON attached ECKD storage
+
+[discrete]
+==== Restrictions
+
+The following restrictions impact {product-title} on {ibmzProductName} and {linuxoneProductName}:
+
+* Automatic repair of damaged machines with machine health checking
+* {openshift-local-productname}
+* Controlling overcommit and managing container density on nodes
+* NVMe
+* OpenShift Virtualization
+* Precision Time Protocol (PTP) hardware
+* Compute nodes must run {op-system-first}
+* Persistent shared storage must be provisioned by using either {rh-storage-first} or other supported storage protocols
+* Persistent non-shared storage must be provisioned using local storage, like iSCSI, FC, or using LSO with DASD, FCP, or EDEV/FBA
+
+[id="ocp-4-13-ibm-power"]
+=== {ibmpowerProductName}
+
+With this release, {ibmpowerProductName} is now compatible with {product-title} {product-version}. For installation instructions, see the following documentation:
+
+* xref:../installing/installing_ibm_power/installing-ibm-power.adoc#installing-ibm-power_installing-ibm-power[Installing a cluster on IBM Power]
+* xref:../installing/installing_ibm_power/installing-restricted-networks-ibm-power.adoc#installing-restricted-networks-ibm-power_installing-restricted-networks-ibm-power[Installing a cluster on IBM Power in a restricted network]
+
+[discrete]
+==== Notable enhancements
+
+The following new features are supported on {ibmpowerProductName} with {product-title} {product-version}:
+
+* Cluster Resource Override Operator
+* Egress IP
+* FIPS cryptography
+* Installer-provisioned Infrastructure Enablement for {ibmpowerProductName} Virtual Server (Technology Preview)
+* MetalLB Operator
+* Network-Bound Disk Encryption - External Tang Server
+
```

Comment:
Add CSI driver for {ibmpowerProductName} Virtual Server

Reply:
@manojnkumar have update the PR fyi. Thanks.

Reply:
@manojnkumar shall I write it as : "{ibmpowerProductName} Virtual Server Block CSI Driver Operator (Technology Preview)" ?

Reply:
Sure.  That sounds fine to me.

Reply:
Tang disk encryption does not work for Power. it should removed from the list.

Reply:
@cs-zhang okay.
Removed `Network-Bound Disk Encryption - External Tang Server` from the `IBM Power notable enhancements` section.
cc : @manojnkumar @prb112

Reply:
@cs-zhang @alishaIBM I think it should remain with a caveat that it is non-multipath disks. It does work on PowerVS, it does not seem to work with multipath disks.

Reply:
Based on our discussion in slack we do support "Network-Bound Disk Encryption - External Tang Server" on Power.
Added it back. Thanks.